### ISSUE TITLE
SAML attributes question

### ISSUE THREAD

**Issue by secure-xxx** [casdoor/casdoor#3338]
Try to pass the custom SAML attribute, for example "memberOf" from ldap synchronized user, but trhough SAML on the another side i catch simple value memberOf, not group list.
<img width="1440" alt="изображение" src="https://github.com/user-attachments/assets/c8c77b34-4ae5-4bad-b255-2ad3410e8ec2">

**Comment by secure-xxx** [comment]
Help me pls, what am i doing wrong? i Try to add custom "filterFields": ["uid", "sAMAccountName", "memberOf"], in ldaps, does not work. How to recieve list of groups as a claim?

**Comment by secure-xxx** [comment]
I tried to add saml parameter in object/saml_idp.go like 3 another (email, name, displayname), but it depends to more things to add, and no luck to compile

**Comment by hsluoyz** [comment]
@secure-xxx we are very hard to follow your issues based on your issue history. Your issues are also very deep into some specific details. I suggest you making the Pull Request by yourself. If you are not capable to do that, let somebody in your company who is capable do that. Closed for now.

**Comment by secure-xxx** [comment]
> @secure-xxx we are very hard to follow your issues based on your issue history. Your issues are also very deep into some specific details. I suggest you making the Pull Request by yourself. It would be a much better ground for us to discuss. If you are not capable to do that, let somebody in your company who is capable do that. Closed for now.

I described each issue in as much detail as possible with a configuration attachment and description. It's more likely that you're on track to close issues per month. I am not a Go programmer and tried to understand the code, but never received a single clear answer to my issues

**Comment by secure-xxx** [comment]
@hsluoyz https://github.com/casdoor/casdoor/pull/3339/commits
